packages/core-data/src/queried-data/reducer.js

```diff
@@ -68,9 +71,9 @@ export function getMergedItemIds( itemIds, nextItemIds, page, perPage ) {
  * @return {Object} Next state.
  */
 function items( state = {}, action ) {
+	const key = action.key || DEFAULT_ENTITY_KEY;
```

Comment:
We can leave this where it was if we have the `DELETE_ITEMS` action contain `itemIds` instead of `items`.